### DISCUSSION TITLE
Custom CTK theme

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [TomSchimansky/CustomTkinter#2079] [Showcase]

**Comment by Akascape** [comment]
@iamironman0 
You can make more themes using https://github.com/avalon60/ctk_theme_builder or https://github.com/Akascape/CTkThemeMaker

**Reply by rudymohammadbali** [reply]
Thanks! Copying and pasting color codes was time-consuming, but now I'll use your app. 👍

**Reply by rudymohammadbali** [reply]
@Akascape 👋 it's seems like you're good at customtkinter framework, and I'm currently working on audio transcriber GUI app using OpenAI Whisper module. If you're interested in working together to make a good quality app let me know.

**Reply by Akascape** [reply]
@iamironman0 Ok, I can work in my free time. I can make the gui for you. Let me know the parameters and its type, and also the input and output forms. If it's working with live audio, we can add a widget like https://github.com/Akascape/TkVisualizer.

BTW, there are many awesome widgets in tkinter, you can explore them here: https://github.com/Akascape/tkinter-toolkit

**Reply by rudymohammadbali** [reply]
Great, here is my discord username:

iamironman_

Add me then I'll give you the information about the app.

**Reply by avalon60** [reply]
Hey @Akascape - If that's what I think it is ([tkinter-toolkit](https://github.com/Akascape/tkinter-toolkit)), i.e. ashowcase for Tkinter widgets, then that's a great idea.

**Reply by avalon60** [reply]
@iamironman0, there are videos available on YouTube showing how to install/use Ctk Theme Builder:
https://www.youtube.com/results?search_query=clive+bostock

**Reply by rudymohammadbali** [reply]
@avalon60 I'm not so noob😂 but thanks

**Reply by avalon60** [reply]
Noob at what? You have used Ctk Theme Builder before then?

**Comment by dipeshSam** [comment]
> https://github.com/iamironman0/Custom-CTK-Theme
> 
> ![6947FABA-B815-40D2-ACF0-BAD64B910FA8_1_105_c](https://github.com/TomSchimansky/CustomTkinter/assets/63475761/f1d1e3c8-2f6a-41f5-a0f8-064f165422a8)
> ![470A32D7-6AD6-4763-92E3-CA9C67B47C7A_1_105_c](https://github.com/TomSchimansky/CustomTkinter/assets/63475761/e8e0ca55-79e0-48ae-977d-b4c21546765b)
> 

**Nice!**

**Reply by rudymohammadbali** [reply]
Glad you like it 👍
